[DOCS] Fix obsolete Slack setup info (#57834)
elastic/elasticsearch: x-pack/docs/en/watcher/actions/slack.asciidoc

You configure the accounts {watcher} can use to communicate with Slack in the
`xpack.notification.slack` namespace in `elasticsearch.yml`.

> You need a Slack App with the Incoming Webhooks feature[https://api.slack.com/messaging/webhooks] 

Reviewer: ```suggestion
You need a Slack App with the https://api.slack.com/messaging/webhooks[Incoming Webhooks feature] 
```

> to configure a Slack account. Use the generated webhook URL to set up your Slack account in Elasticsearch.

Reviewer 2: ```suggestion
You need a Slack App with the Incoming Webhooks feature[https://api.slack.com/messaging/webhooks] 
to configure a Slack account. Use the generated webhook URL to set up your Slack account in {es}.
```

> You need a Slack App with the https://api.slack.com/messaging/webhooks[Incoming Webhooks feature] 
> to configure a Slack account. Use the generated webhook URL to set up your Slack account in Elasticsearch.

Reviewer 2: ```suggestion
You need a Slack App with the https://api.slack.com/messaging/webhooks[Incoming Webhooks feature] 
to configure a Slack account. Use the generated webhook URL to set up your Slack account in {es}.
```

          mrkdwn_in: "pretext, text"
--------------------------------------------------

> To notify multiple channels, create a webhook URL for each channel in Slack 
> and multiple Slack accounts in Elasticsearch (one for each webhook URL).

Reviewer 2: ```suggestion
To notify multiple channels, create a webhook URL for each channel in Slack 
and multiple Slack accounts in {es} (one for each webhook URL).
```
